test/e2e/support/utils.js

```diff
@@ -90,7 +90,34 @@ export async function newDesktopBrowserPage() {
 		fail( error );
 	} );
 
-	await page.setViewport( { width: 1000, height: 700 } );
+	await setDesktopViewport();
+}
+
+export async function setDesktopViewport() {
```

Comment:
Can we generalize this method as follows:

```js
export async function setViewport( type ) {
    const allowedDimensions = { desktop: { width: 1000, height: 700 } };
    const currentDimmension = allowedDimensions[ type ];
    await page.setViewport( currentDimmension );
    await waitForPageDimensions( currentDimmension.width, currentDimmension.height );
}
```
I'm wondering if also should align the names with what we use for the `viewport` module.

Reply:
Feedback was addressed. Than you for the review!